demosplan/DemosPlanCoreBundle/Logic/Faq/FaqService.php

```diff
@@ -144,19 +156,30 @@ public function getEnabledAndDisabledFaqList(FaqCategory $faqCategory): array
     /**
      * Get enabled FAQs of a given category.
      *
-     * @return array<int, Faq>
+     * @return array<int, FaqInterface>
      */
-    public function getEnabledFaqList(FaqCategory $faqCategory, User $user): array
+    public function getEnabledFaqList(FaqCategoryInterface $faqCategory, User $user): array
     {
         $roles = $user->isPublicUser() ? [Role::GUEST] : $user->getRoles();
+        $categoryName = '';
+        $className = '';
+        if ($faqCategory instanceof FaqCategory) {
+            $categoryName = 'faqCategory';
+            $className = Faq::class;
+        }
+        if ($faqCategory instanceof PlatformFaqCategory) {
+            $categoryName = 'platformFaqCategory';
+            $className = PlatformFaq::class;
+        }
+
```

Comment:
Minor: The logic can be reduce a little bit:

```suggestion
        $categoryName = 'faqCategory';
        $className = Faq::class;

        if ($faqCategory instanceof PlatformFaqCategory) {
            $categoryName = 'platformFaqCategory';
            $className = PlatformFaq::class;
        }
```